chart/docs/values-config.md

```diff
@@ -0,0 +1,128 @@
+# Tobs helm values config
```

Comment:
hmm. I think there are two different types of config here.
1. config which you need only for things that don't use the CLI (high-availability)
2. config applicable to both helm chart and CLI. (multicluster)

Type 1 is ok here. I'd move type 2 stuff into a new top-level /docs directory. It does not belong under /chart and I'd add new links from the front-page readme here.

Reply:
So currently using CLI we offer only abstract features with flags like `HA`, `integration with forge`, etc... all these features can also be configured within `values.yaml`. All the flexibility lies in the `values.yaml` so the intention behind creating this file is to document all possible configurations across different components in tobs.

Yes, I agree with you only config related to `values.yaml` will reside in this file. Anything about CLI will be in a new file in the high level root directory of this repo.